src/main/scala/com/databricks/labs/mosaic/functions/MosaicContext.scala

```diff
@@ -939,7 +939,12 @@ object MosaicContext extends Logging {
     def checkDBR(spark: SparkSession): Boolean = {
         val sparkVersion = spark.conf.get("spark.databricks.clusterUsageTags.sparkVersion", "")
         val isML = sparkVersion.contains("-ml-")
-        val isPhoton = spark.conf.get("spark.databricks.photon.enabled", "false").toBoolean
+        var isPhoton = false
+        try {
```

Comment:
@edurdevic can we replace this try block with `getOption` call:
`spark.conf.getOption("spark.databricks.photon.enabled").getOrElse("false").toBoolean`